### PR TITLE
Updating README to reflect latest version of Core

### PR DESCRIPTION
**Pull request by ericthompson**

### CODE REVIEW

**Comment by Jahnp** [comment]
Approved

File: README.md [hunk 1]
```diff
@@ -4,9 +4,7 @@
 
 Fabric JS is a JavaScript front-end framework which provides visuals-focused [components](https://github.com/OfficeDev/office-ui-fabric-js/blob/master/ghdocs/COMPONENTSLIST.md) to extend, re-work and use in your Office and Office 365 experiences. Fabric JS uses styles from the [Office UI Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core/) project. Fabric JS is a collection of responsive, mobile-first styles and tools designed to make it quick and simple for you to create web experiences using the Office Design Language. 
 
-**Latest version:** Fabric JS 1.2.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 4.0.0.
-
-**Latest version on CDN:** Fabric JS 1.0.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 4.0.0.
+**Latest version:** Fabric JS 1.2.0, which uses [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) 5.0.1.
```
Comment (mikewheaton) [inline README.md]: Actually, looks like 1.2.0 still uses Fabric Core 4.0.0. This will be bumped ahead in the next release.

Comment (mikewheaton) [inline README.md]: As part of this PR let's also update the CDN section to point people to use https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.2.0/css/fabric.min.css as the reference for Fabric Core. This way they don't have to know which version of Core is bundled and there's less chance for bugs.

Comment (ericthompson) [inline README.md]: Sure thing - PR coming shortly.

**Comment by mikewheaton** [comment]
Approved! 😋

**Pull request merged**